docs/policies.md

```diff
@@ -0,0 +1,139 @@
+# Ansible Collection Policies
+
+This doc explains the necessary ansible-build-data changes to enforce
+Ansible Community Steering Committee Policies such as [Removal from Ansible][1]
+and [Repository management][2].
+
+## Removal from Ansible
+
+TODO
+
+## Repository management
+
+### Background
+
+From ansible 7.2.0 onwards, each release contains a `ansible-VERSION-tags.yaml`
```

Comment:
Is it intentional you use ansible in lower-case many times? When it's clear that the package name is meant I think this is OK, but here it sounds more like the product, and that shoul be Ansible.

Reply:
I use lower case ansible to refer to the ansible community package.

Reply:
I assumed so. Here it could be both, that's why I was asking :)